src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -32,10 +32,10 @@ export function ThemeProvider({
     [customProperties, themeConfig, unstableGlobalTheming],
   );
 
-  // We want these values to be `null` instead of `undefined` when not set.
+  // We want these values to be empty string instead of `undefined` when not set.
   // Otherwise, setting a style property to `undefined` does not remove it from the DOM.
-  const backgroundColor = customProperties['--p-surface-background'] || null;
-  const color = customProperties['--p-text-on-surface'] || null;
+  const backgroundColor = customProperties['--p-surface-background'] || '';
```

Comment:
backgroundColor is now not nullable. color can be nullable but i figured we might as well make this consistent as it seems to behave the same in chrome for me. @tmlayton  can you confirm the intended behaviour still works for you?

Reply:
If the 🎩 looks good I’m fine with it. Try toggling global theming on and off and make sure it works.